### PR TITLE
EZP-28067; EZP-29713: Implement bulk delete, bulk move - pass udwConfig to Sub-items

### PR DESCRIPTION
**Pull request by tischsoic**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28067 https://jira.ez.no/browse/EZP-29713
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Needed for https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/107
As discussed: https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/107#discussion_r228430123

**Should be merged after:** https://github.com/ezsystems/ezplatform-admin-ui/pull/691

Why another PR? Because features in https://github.com/ezsystems/ezplatform-admin-ui/pull/691 are very generic and support generic bulk. This ticket and  https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/107 are meant to support only bulk move + move to trash for Sub-items.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/locationview.html.twig [hunk 1]
```diff
@@ -66,7 +66,12 @@
                     <div class="container px-5">
                         <section class="ez-fieldgroup">
                             <h2 class="ez-fieldgroup__name">{{ 'content.view.subitems'|trans|desc('Sub-items') }}</h2>
-                            <div class="ez-sil" data-location="{{ location.id }}" data-items="{{ subitems_module.items }}" data-content-types="{{ subitems_module.content_type_info_list }}"></div>
+                            <div class="ez-sil" 
+                                data-location="{{ location.id }}" 
+                                data-items="{{ subitems_module.items }}" 
+                                data-content-types="{{ subitems_module.content_type_info_list }}"
+                                data-udw-config="{{ ez_udw_config('single', {}) }}"
```
Comment (dew326) [inline src/bundle/Resources/views/content/locationview.html.twig]: `single_container` not `single`
And I was thinking about the name of a data-attribute. What if we would need to pass another configuration of UDW in other button?
I think we need to call it more precise like `data-move-udw-config`.

**Comment by tischsoic** [comment]
In case of review, review only changes from: https://github.com/ezsystems/ezplatform-admin-ui/pull/702/commits/f656f6773ebfb427ec7d854d3fed5559b22142ec

File: src/bundle/Resources/views/content/locationview.html.twig (hunk 1 above)
Comment (tischsoic) [reply src/bundle/Resources/views/content/locationview.html.twig]: Fixed: https://github.com/ezsystems/ezplatform-admin-ui/pull/702/commits/19163722885b5a1462e1886515e691f6143ef6eb

**Comment by tischsoic** [comment]
ping @dew326

File: src/bundle/Resources/public/js/scripts/admin.location.view.js [hunk 2]
```diff
@@ -69,6 +69,7 @@
 
             return total;
         }, {});
+        const udwConfigMove = JSON.parse(container.dataset.udwConfigMove);
```
Comment (sunpietro) [inline src/bundle/Resources/public/js/scripts/admin.location.view.js]: `udwConfigMove` means nothing to me. It's lacking the last part, in my opinion. Is it move to trash or move location?

Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/admin.location.view.js]: Is it for bulk move?

Comment (tischsoic) [reply src/bundle/Resources/public/js/scripts/admin.location.view.js]: Yes

Comment (sunpietro) [reply src/bundle/Resources/public/js/scripts/admin.location.view.js]: Then  maybe `udwConfigBulkMoveItems`

Comment (tischsoic) [reply src/bundle/Resources/public/js/scripts/admin.location.view.js]: Works for me. :)

**Comment by tischsoic** [comment]
ping @sunpietro

**Comment by micszo** [comment]
Issue with date on travis occurred also in nightly build.

**Pull request merged**